test/test_component.py

```diff
@@ -291,17 +293,22 @@ def test_high_level_constructors(self):
         doc.add(media_comp)
         assert doc_diff(doc, hlc_doc) == 0, f'Constructor Error: {media_identity}'
 
-    def test_sep055(self):
+    def test_restriction_enzyme_bp011(self):
         """Test construction of components and features using helper functions: for each, build manually and compare."""
         hlc_doc = sbol3.Document()
```

Comment:
Why are these docs here? They are not getting used.

Reply:
True, they are not getting used It went there becasue I copied the whole start of the test func. I removed some extra stuff that I noticed in other parts as well.